### DISCUSSION TITLE
httpx Client documentation suggests realm is supported, but it is ignored.

### DISCUSSION THREAD

**Discussion by schinckel** [encode/httpx#1586] [Potential Issue]
The documentation for `authlib.integrations.httpx_client.OAuth1Client` suggests it supports passing in a `realm` argument, but it actually calls it `OAuth1Session`, which probably means it is a copy-paste from the requests section.

https://docs.authlib.org/en/latest/client/api.html?highlight=realm#authlib.integrations.httpx_client.OAuth1Client.fetch_request_token

Anyway, this value is not passed through to the underlying `OAuth1Auth` object, meaning my authentication was failing.

(My workaround is to set the `client.auth.realm` after instantiation, but it's still a bit of a PITA).

**Comment by lovelydinosaur** [comment]
Okey dokes. I guess you oughta open this against the `authlib` GitHub repo, right?
Or is there something in our docs that we need to change too?

**Reply by schinckel** [reply]
Ah, yeah, seems like I clicked on the wrong repo link.
